### ISSUE TITLE
MD038: Returning false positives for empty strings

### ISSUE THREAD

**Issue by sujayhazrashell** [markdownlint/markdownlint#504]
## Description
The linting rule throws error for empty strings as 
**_MD038/no-space-in-code Spaces inside code span elements [Context: " ```"]_**

The line of code for which this error is thrown is:
<```>

### Environment
I am using .markdown-lint.yml

### Expected Behavior
Markdown linting should happen successfully without throwing any error

### Actual Behavior
Linting fails with the error message 
**MD038/no-space-in-code Spaces inside code span elements [Context: " ```"]**

## Replication Case
Sample code from the markdown where the error is thrown 
??? note annotate "Bash Terminal"
    === "1 - Clone the repo"
        - Open your terminal.
        - Clone the repo to your local.
        ```abc
        git clone <your-repo-name-here>
        cd <your-repo-name-here>
        ``` -> this is where the error is raised

**Comment by DavidAnson** [comment]
Are you sure you are using the Ruby implementation of markdownlint? That violation looks like what the Node implementation would generate. However, the sample you provide does not produce any violations:

https://dlaa.me/markdownlint/#%25m%23%20Issue%20504%0A%0A%3C%60%60%60%3E%0A

**Comment by jaymzh** [comment]
I cannot reproduce this, I suspect you filed this against the wrong markdownlint
